src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -1,11 +1,7 @@
 package com.nirima.jenkins.plugins.docker;
 
 import com.github.dockerjava.api.command.CreateContainerCmd;
-import com.github.dockerjava.api.model.Bind;
-import com.github.dockerjava.api.model.PortBinding;
-import com.github.dockerjava.api.model.Volume;
-import com.github.dockerjava.api.model.VolumesFrom;
-import com.github.dockerjava.api.model.Device;
+import com.github.dockerjava.api.model.*;
```

Comment:
Opinions vary on this subject but, personally, I'm not a fan of `.*` in imports, and I'm even less of a fan of unnecessary changes to source code.
Please just add the imports you need, one per line.